server/gendb/gendb.go

```diff
@@ -37,6 +37,9 @@ type XDBConfig struct {
 	// DynamoDB configuration.
 	DynamoDB AWSDynamoDBTable
 
+	// EnableTransactions determines which schemas this schema will be able to perform transactions with. It only needs to be set for one per pair.
```

Comment:
The order of items in a transaction doesn't matter, right? At least in terms of the semantics. (I think maybe if a transaction is going to fail, it might use less read/write capacity on the DB if it fails sooner?)

Reply:
Also, this should work if you put a table to transact with itself?

Reply:
1) The order of the items doesn't matter in regards to semantics and might not even matter when it comes to the underlying db calls, but I'm not certain about that. This line `DynamoDB performs two underlying reads or writes of every item in the transaction: one to prepare the transaction and one to commit the transaction. ` from the docs makes it sound like regardless of the format of the transaction, the underlying read/writes happen in two steps so it seems that the first step is the condition check(s) that all happen regardless of if one fails, and the second write step that may consume the same capacity regardless of failure? Not certain though

2) Correct! With this setup the table can transact with itself